### PR TITLE
[DUOS-2789] Switch pages and components to JSX

### PR DESCRIPTION
**Pull request by fboulnois**
### Addresses

https://broadworkbench.atlassian.net/browse/DUOS-2789

### Summary

Switch pages and components to JSX

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/components/PageHeading.js [hunk 1]
```diff
@@ -1,82 +1,80 @@
...
+  if (iconSize === 'none') {
+    return NONE;
+  } else if (iconSize === 'large') {
+    return LARGE;
+  } else {
+    return MEDIUM;
   }
-
-});
+}
```
Comment (rushtong) [inline src/components/PageHeading.js]: I see a number of minor eslint warnings starting here:
```
WARNING in [eslint] 
src/components/PageHeading.js
  Line 25:2:  Missing semicolon  @babel/semi
  Line 78:2:  Missing semicolon  @babel/semi

src/components/PaginationBar.js
  Line 33:1:  Expected indentation of 12 spaces but found 16  indent
  Line 34:1:  Expected indentation of 12 spaces but found 16  indent
  Line 41:1:  Expected indentation of 12 spaces but found 17  indent
  Line 42:1:  Expected indentation of 12 spaces but found 17  indent
  Line 43:1:  Expected indentation of 12 spaces but found 17  indent
  Line 44:1:  Expected indentation of 12 spaces but found 17  indent
  Line 49:1:  Expected indentation of 12 spaces but found 16  indent
  Line 50:1:  Expected indentation of 12 spaces but found 16  indent
  Line 58:1:  Expected indentation of 10 spaces but found 15  indent
  Line 59:1:  Expected indentation of 10 spaces but found 15  indent
  Line 60:1:  Expected indentation of 10 spaces but found 15  indent
  Line 61:1:  Expected indentation of 10 spaces but found 15  indent

src/components/ReadMore.js
  Line 25:4:  Missing semicolon  @babel/semi
  Line 38:4:  Missing semicolon  @babel/semi
  Line 42:4:  Missing semicolon  @babel/semi
  Line 46:4:  Missing semicolon  @babel/semi
  Line 50:4:  Missing semicolon  @babel/semi
  Line 54:4:  Missing semicolon  @babel/semi

src/components/SignIn.js
  Line 171:13:  Missing semicolon  @babel/semi
  Line 191:2:   Missing semicolon  @babel/semi
```

**Review by rushtong — APPROVED** [review]
Aside from the lint warnings, changes look good!

**Review by rushtong — APPROVED** [review]
Thank you! 👍🏽

**Review by hams7504 — APPROVED** [review]
looks great!

**Pull request merged**